nhd/Node.py

```diff
@@ -260,10 +260,13 @@ def GetFreeNumaNicResources(self) -> List[int]:
         ninfo = [[] for _ in range(self.numa_nodes)]
 
         for n in self.nics:
-            if ENABLE_SHARING:
-                ninfo[n.numa_node].append([n.speed*NIC_BW_AVAIL_PERCENT - n.speed_used[x] for x in range(2)])
-            else:
-                ninfo[n.numa_node].append([0 if (n.pods_used > 0) else n.speed*NIC_BW_AVAIL_PERCENT for x in range(2)])
+            try:
```

Comment:
Is this due to a configuration file error?

Reply:
No, they added some new nodes to the PP cluster today, and NHD was throwing an exception here regarding the NICs.